config/nrfconnect/chip-module/Kconfig

```diff
@@ -70,3 +70,218 @@ config CHIP_DEBUG_SYMBOLS
+
+config CHIP_FACTORY_DATA_GENERATE_PAI_CERT
+	bool "Enables generation PAI cert after build"
+	help
+	  Enables generation Product Attestation Intermediate Certificate 
+	  according to given Vendor and Product IDs.
+	  To generate PAI Cert a  a chip-cert executable must be available 
+	  from system variables environment PATH.
+	  Currently only on Linux system PAI cert generation is supported.
+
+if !CHIP_FACTORY_DATA_GENERATE_DAC_CERT && !CHIP_FACTORY_DATA_GENERATE_PAI_CERT
+
+# Certificates auto generation
+config CHIP_FACTORY_DATA_USE_DEFAULTS_CERTS_PATH
+	bool "Uses a default certificates located in Matter repository"
+	default y
+	help
+	  Pre-generated certificates can be used for development porpose.
+	  This config includes default pre-generated certificates 
+	  which are located in credentials/development/attestation/ directory 
+	  instead of generate new ones.
+	  This option can be enabled only if CHIP_FACTORY_DATA_GENERATE_DAC_CERT
+	  and CHIP_FACTORY_DATA_GENERATE_PAI_CERT are set to n.
+
+endif # !CHIP_FACTORY_DATA_GENERATE_DAC_CERT && !CHIP_FACTORY_DATA_GENERATE_PAI_CERT
+
+# Configs for SPAKE2 generation
+config CHIP_FACTORY_DATA_GENERATE_SPAKE2_VERIFIER
+	bool "Enables generation spake2 verifier"
+	help
+	  Enables generation spake2 verifier according to 
+	  given iteration counter, salt and passcode.
+	  To generate Spake2 verifier a spake2p executable must be available 
+	  from system variables environment.
+	  Currently only Linux system is supported to generate PAI cert
+
+config CHIP_DEVICE_DAC_CERT_PATH
+	string "A path to Device Attestation Certificate"
+	depends on !CHIP_FACTORY_DATA_GENERATE_DAC_CERT
+	help
+	  A full path to Device Attestation Certificate file in .der format or
+	  raw data of certifiate. Certificate can be generated during build 
+	  if CHIP_FACTORY_DATA_GENERATE_DAC_CERT is enabled
+
+config CHIP_DEVICE_DAC_KEY_PATH
+	string "A path to Device Attestation Certificate Key"
+	depends on !CHIP_FACTORY_DATA_GENERATE_DAC_CERT
+	help
+	  A full path to Device Attestation Certificate Key file in .der format or
+	  raw data of certificate key. Key for this certificate can be generated during build 
+	  if CHIP_FACTORY_DATA_GENERATE_DAC_CERT is enabled
+
+config CHIP_DEVICE_PAI_CERT_PATH
+	string "A path to Product Attestation Intermediate"
+	depends on !CHIP_FACTORY_DATA_GENERATE_PAI_CERT
+	help
+	  A full path to Product Attestation Intermediate file in .der format or
+	  raw data of certifiate. Certificate can be generated during build 
+	  if CHIP_FACTORY_DATA_GENERATE_DAC_CERT is enabled
+
+endif #CHIP_FACTORY_DATA_ENABLE
+
+# Factory data parameters
+config CHIP_DEVICE_SERIAL_NUMBER
+	string "Serial number of device"
+	default "11223344556677889900"
+	help
+	  Serial number of a device which can be used to identify
+	  the serial number field in the Matter certificate structure. 
+	  Maximum length of serial number is 20 bytes.
+	  Strings longer than 20 bytes will be declined in script.
```

Comment:
The same comment, those are generic parameters used for various purposes in the Matter stack, so we should not put comments applying to certificate and scripts only.